bot/constants.py

```diff
@@ -501,6 +510,30 @@ class RedirectOutput(metaclass=YAMLGetter):
     delete_delay: int
 
 
+class Event(Enum):
```

Comment:
Why was this moved?

Reply:
kaizen. It was in a stupid place, and belongs with the other classes.